_posts/2021-01-13-voicemeeter-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,385 @@
+
+Image for showing where the menu is:
+{
+
+
+
+  Need Picture here
+
+
+
+}
+
+Image for showing Save / Load / Reset Settings:
+{
+
+
+
+  Need Picture here
+
+
+
+}
+
+Voicemeeter Banana is likely flashing in the top right telling you to set up a hardware output? For good reason! you can't hear anything you are adjusting if you haven't set it up. This is ok and you should go ahead and do this. One of the Three Hardware Outputs should be your listening device (headphones, speakers, other). This is so that you wil be able to hear things once we set audio to run through voicemeeter. Another one should also be a virtual audio cable that you may have installed, it could be named such as 'VB Audio Cable'. What will we be doing with this virtual audio device? We will be sending our stream audio to it!
+
+Oh, you might have noticed that the list that pops up is repetitive with some minor variances? That is because on your computer there are different drivers which can be interacted with. In some cased WDM type drivers work better, in others MME type drivers work better. I suggest starting with WDM. If that turns out wonky you can try MME. I might cover more on the differences later but for now lets just try WDM to start! Alright, that will be all for this part, your hardware outputs should look a little something like this:
+
+Banana Hardware Outs Setup:
+{
+
+
+
+  Need Picture here
+
+
+
+
+}
+
+You know what we need now? Some audio to thread into these outputs! Thats what! When do we want this audio? We Want this audio yesteryear! So lets get some of that audio. In windows we need to open sound settings, it doesn't matter how we get there, whether you search in the start menu or right click the speaker icon in the task tray thing likely on the bottom right of windows unless you moved your task bar. If right clicking on the speaker icon it should be called 'Open Sound Settings' and if going through the Start Menu it will be just 'Sound Settings'. Under output we will choose our output device, this is our main output of all audio. Lets change this to 'Voicemeeter VAIO'. You may notice that if you had audio going through your listening device it likely disapeared. This is because it is now routed to the Virtual Input Voicemeeter VAIO. If you look inside Voicemeeter Banana, you should now see the audio bar boincing if you are playing audio. Do you want to hear it? Tak ea look at A1-3 and toggle the one that matches the A beside your headphones/speakers.
```

Comment:
Instead of suggesting them to pick any audio output, tell them a specific output to use for their first route. Handhold the user extensively their first time through because the slightest deviation will cause them to give up.

Reply:
True, a small problem would be my naming might be off as I've messed with the drivers, I could install banana on a different computer to get the default naming schemes though. That would probably be best.